src/cosima/src/MCActivator.cc

```diff
@@ -423,33 +430,38 @@ bool MCActivator::CalculateEquilibriumRates()
               G4Ions* Nucleus = dynamic_cast<G4Ions*>(Tree[b].back().GetDefinition()); 
               
               bool LevelsOK = true;
-              G4NuclearLevelManager* M = G4NuclearLevelStore::GetInstance()->GetManager(Nucleus->GetAtomicNumber(), Nucleus->GetAtomicMass());
-              if (M->IsValid() == true) {
+              //G4NucLearLevelManager* M = G4NucLevelStore::GetInstance()->GetManager(Nucleus->GetAtomicNumber(), Nucleus->GetAtomicMass());
+              const G4LevelManager* M = G4NuclearLevelData::GetInstance()->GetLevelManager(Nucleus->GetAtomicNumber(), Nucleus->GetAtomicMass());
+	      
+	      //if (M->IsValid() == true) {
```

Comment:
the class `G4LevelManager` has no more a method like `IsValid()` so for now i put the statement `IsValid` always true. Not sure if it is correct.